packages/cli/src/utils/git.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as fs from 'fs/promises';
+import path from 'path';
+
+async function findGitRoot(startDir: string): Promise<string | null> {
```

Comment:
I would think we have something similar already in the core package. Could be worth checking to avoid creating a duplicate helper.